### PR TITLE
Rename Microsoft.AspNetCore.App.PlatformManifest.txt 

### PR DESCRIPTION
**Pull request by natemcmaster**
Fix https://github.com/aspnet/AspNetCore/issues/8836

#### Description
The SDK is not reading version-conflict resolution data from our targeting pack because the manifest is in the wrong location. As a result, the SDK is copying assemblies into the build/publish output which should have come from the shared framework instead.
	
#### Customer Impact
When a project references EF Core 3.0 and ASP.NET Core 3.0,  published output is not optimized and duplicates assemblies which should have come from the shared framework.

#### Regression?
Yes, this is a regression from the shared framework behavior in 2.2.

#### Risk
Low.

### CODE REVIEW

**Review by dsplaisted — APPROVED** [review]

**Review by JunTaoLuo — APPROVED** [review]

**Review by dougbu — APPROVED** [review]

**Comment by JunTaoLuo** [comment]
@Eilon for QB process

**Comment by natemcmaster** [comment]
Tracking QB approval here: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/843061/

**Review by Eilon — APPROVED** [review]
Change looks good, approved by me, but please wait for QB approval in AzDO.

**Comment by natemcmaster** [comment]
QB approved, waiting for a re-run of Templates Tests before I merge (first run failed with yarn flakiness).

**Comment by Eilon** [comment]
**QB approved for Preview 4. Please check in and watch the build stay green.**

And please also resolve https://devdiv.visualstudio.com/DevDiv/_workitems/edit/843061 when done.

**Pull request merged**
